README.md

```diff
@@ -90,14 +90,12 @@ Then create the config file `etc/config.yaml`. Here is an example your could sta
 ---
 tenants:
   - index: monocle
-    crawler:
-      loop_delay: 10
-      github_orgs:
-        - name: tektoncd
-          repository: pipeline
-          updated_since: "2020-05-01"
-          token: <github_token>
-          base_url: https://github.com
+    crawlers:
+      - name: github-tektoncd
+        provider:
+          github_token: <github_token>
```

Comment:
github_url is optional now, but it seems the legacy code do not provide a fallback to github.com

Reply:
Ok saw it later in the migration tool